model/command_request.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import (
+	"encoding/json"
+	"io"
+)
+
+type CommandMoveRequest struct {
+	TeamId string `json:"team_id"`
+}
+
+func TeamIdFromCommandMoveRequestJson(data io.Reader) (string, error) {
```

Comment:
I recommend being consistent with how other `<Struct>FromJson` are named, and it's possible if we ever expand this struct that someone will come around and have another `<Prop>FromCommandMoveRequestJson` func :)

I recommend `func CommandMoveRequestFromJson(data io.Reader) (*CommandMoveRequest, error) {`

Reply:
Addressed via https://github.com/mattermost/mattermost-server/pull/13624/commits/d0222d7d355c15d61b483322efaf3d83baa95d3a